### DISCUSSION TITLE
Allow "save and stay" to be the default save button action

### DISCUSSION THREAD

**Discussion by alexchopin** [directus/directus#24897] [Feature Requests]
### Discussed in https://github.com/directus/directus/discussions/4081

<div type='discussions-op-text'>

<sup>Originally posted by **jordie23** February 16, 2021</sup>
I'm currently evaluating Directus CMS for our site. 

While testing, I've noticed the save button on content is "save and return to list". I know this will annoy our writers, is there a way to default it to "save and stay"? I know there's command+s, but not all our writers are the most tech-aware and would opt for the button still. 

If it's not a feature already (couldn't see it anywhere) then I'd like to request it. Perhaps it could be a user preference option?</div>

**Comment by sbuchweitz** [comment]
I too think this functionality is needed. Default users are annoyed by item closing every time when clicking save

**Comment by JamesW1** [comment]
Thanks for the suggestion! It is something that we expect to be resolved as part of providing a persistent draft/published state, which would introduce auto-saving. 

This is something that is already planned, trackable here: https://roadmap.directus.io/roadmap?id=ac86b729-311a-4a6c-b52c-2bd4504d19b0
